src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -32,5 +34,57 @@ export class OverflowMenuPane extends Dialog {
 		 * so we need to add some compensation)
 		 */
 		this.addGap["bottom"] = pos => position.addOffset(pos, -20, 60);
+
+		setTimeout(() => this.listItems()[0].focus());
+	}
+
+	@HostListener("keydown", ["$event"])
+	hostkeys(event: KeyboardEvent) {
+		this.escapeClose(event);
+		const listItems = this.listItems();
+
+		switch (event.key) {
+			case "ArrowDown":
```

Comment:
Any other IE11 specific values? Like `"Down"`?

Reply:
Theres `"Esc"` as well. Home and End should work on IE11